src/test/java/athleticli/ui/dietgoal/DietGoalTest.java

```diff
@@ -15,12 +15,27 @@ void getNutrients_initialiseCommonArgs_expectArgs() {
         assertEquals("protein", proteinGoal.getNutrients());
     }
 
+    @Test
+    void setNutrients_setCommonArgs_expectArgs() {
+        DietGoal proteinGoal = new DietGoal("protein", 10000);
+        proteinGoal.setNutrients("Advanced Protein");
+        assertEquals("Advanced Protein", proteinGoal.getNutrients());
+    }
+
     @Test
     void getTargetValue_initialiseCommonArgs_expectArgs() {
         DietGoal proteinGoal = new DietGoal("protein", 10000);
         assertEquals(10000, proteinGoal.getTargetValue());
     }
 
+    @Test
+    void setTargetValue_initialiseCommonArgs_expectArgs() {
+        DietGoal proteinGoal = new DietGoal("protein", 10000);
+        proteinGoal.setTargetValue(10);
+        assertEquals(10, proteinGoal.getTargetValue());
+
+    }
+
     @Test
     void getCurrentValue_initialiseCommonArgs_expectZero() {
```

Comment:
According to the [coding standard](https://se-education.org/guides/conventions/java/basic.html#comments), we are recommended to use American spelling.
```suggestion
    void getCurrentValue_initializeCommonArgs_expectZero() {
```